include/DataFile.h

```diff
@@ -127,8 +127,8 @@ class LMMS_EXPORT DataFile : public QDomDocument
 	void upgrade_mixerRename();
 	void upgrade_bbTcoRename();
 	void upgrade_sampleAndHold();
-  void upgrade_midiCCIndexing();
 	void upgrade_loopsRename();
+  	void upgrade_midiCCIndexing();
```

Comment:
I believe I actually did tell you earlier to move it to after `upgrade_sampleAndHold()`. I should have been more clear. I meant to move it to the end of the list but now the code has changed so the end of the list is after `upgrade_midiCCIndexing`. I believe this to be the correct place of new upgrade routines but I'm not absolutely sure. I will bump the developers on discord about this. I do think a3cad7c should be reverted however, apart from the fix of spaces to tabs in front of `void upgrade_midiCCIndexing();`.

Reply:
Update. I got a confirm that the new routine should be placed last in line.